### DISCUSSION TITLE
Support for simultaneous events?

### DISCUSSION THREAD

**Discussion by studiokrause** [cpvalente/ontime#743] [Ideas]
Do you plan in any form support for the events played in the same time? I often stream city festivals with multiple scenes/subevents and it would be ideal to have tool for showing (or even planing) the cameras and reporters across these events.

**Comment by cpvalente** [comment]
Hey @studiokrause, I've been putting off replying, but I think you deserve an answer.

Quick clarification:

Do you mean that several unrelated events are happening at the same time? 
ie: Fatboy Slim on stage 1 and Jay Z on stage 2

or that there are several things happening in one event at the same time?
ie: Camera 1: Cut 5, Video: FX 3
